src/internal.h

```diff
@@ -1053,7 +1053,7 @@ _dispatch_ktrace_impl(uint32_t code, uint64_t a, uint64_t b,
 #else
 #define _dispatch_client_assert_fail(fmt, ...)  do { \
 		char *_msg = NULL; \
-		asprintf(&_msg, "%s" fmt, DISPATCH_ASSERTION_FAILED_MESSAGE, \
+		(void)asprintf(&_msg, "%s" fmt, DISPATCH_ASSERTION_FAILED_MESSAGE, \
```

Comment:
For completeness, the same change should be made to the snprintf() call in the _WIN32 branch.

Reply:
Updated as requested. Thanks for being more alert than me, I only looked for more asprintf instances.